music_album.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'item_music'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :release_year, :artist
+
+  def initialize(title, artist, release_year)
+    super(title)
```

Comment:
@ShaliniNayan MusicAlbum extend Items cass from item.rb, we don't have to create another class such as item from item_music.rb